e2e/api_test.go

```diff
@@ -272,12 +272,14 @@ func TestE2E_TaskEndpoints_UpdateEnableDisable(t *testing.T) {
 
 	cts := ctsSetup(t, srv, tempDir, disabledTaskConfig(tempDir))
 
-	// Confirm that terraform files were not generated for a disabled task
-	files := testutils.CheckDir(t, true, fmt.Sprintf("%s/%s", tempDir, "disabled_task"))
-	require.Equal(t, len(files), 0)
+	// Confirm that terraform files were generated for a disabled task
+	taskPath := filepath.Join(tempDir, disabledTaskName)
+	files := testutils.CheckDir(t, true, taskPath)
+	require.Greater(t, len(files), 0)
+	testutils.CheckFile(t, true, taskPath, "terraform.tfvars.tmpl")
```

Comment:
nice! this was the missing ingredient. In the future this would be a good check to see that a task actually ran. Maybe in the future we could make this into a utils call? Something like "testutils.VerifyTaskRan" which could do the event checking and and check for the terraform.tfvars.tmpl file. Something like that.

Reply:
Yeah, this in particular could be more of a check task is initialized rather than it ran. But it would be a good validation method since we'll have more situations where a task is initialized but not run!

Reply:
Will punt this to a later date!